pyaedt/modeler/cad/Modeler.py

```diff
@@ -2526,15 +2495,11 @@ def convert_to_selections(self, object_id, return_list=False):
            Name of the objects corresponding to the one or more object IDs passed as arguments.
 
         """
-        if "netref.builtins.list" in str(type(object_id)):
-            list_new = []
-            for i in range(len(object_id)):
-                list_new.append(object_id[i])
-        elif not isinstance(object_id, list):
+        if not isinstance(object_id, list):
             object_id = [object_id]
         objnames = []
         for el in object_id:
-            if isinstance(el, int) and el in list(self.objects.keys()):
+            if isinstance(el, int) and el in self.objects:
                 objnames.append(self.objects[el].name)
             elif isinstance(el, int):
                 objnames.append(el)
```

Comment:
why there is this case? The returned list should contain the object names, here it is putting in an id.